src/apps/intel_mp/intel_mp.lua

```diff
@@ -569,12 +570,14 @@ function Intel:push ()
    if not self.txq then return end
    local li = self.input["input"]
    if li == nil then return end
---   assert(li, "intel_mp:push: no input link")
 
    while not empty(li) and self:ringnext(self.tdt) ~= self.tdh do
       local p = receive(li)
-      -- NB: see comment in intel10g for why this is commented out,
-      --     the rest of the loop body goes in an else branch
+      -- NB: the comment below is taken from intel_mp.lua, which disables
```

Comment:
Should this say `intel10g` instead of `intel_mp.lua`?

Reply:
Yes you are right.  Will fix in a followup.  Thanks for the review!